README.md

```diff
@@ -178,4 +180,4 @@ The IDB is not responsible, under any circumstance, for damage or compensation,
 
 I. Under any concept of intellectual property, negligence or detriment of another part theory; I
 
-ii. Following the use of the Digital Tool, including, but not limited to defects in the Digital Tool, or the loss or inaccuracy of data of any kind. The foregoing includes expenses or damages associated with communication failures and / or malfunctions of computers, linked to the use of the Digital Tool.
```

Comment:
There's no change here, but using the GitHub editor (which I'm restricted to, at the moment) I'm unable to prevent this diff. May be a line ending change.